documentation/onboarding.md

```diff
@@ -16,7 +16,67 @@ Please read the [ARIA WG Process Document](process.md)
 * add **Channels**, "#aria"
 
 ### IRC command cheat sheet
-* [ARIA Issue 1404](https://github.com/w3c/aria/issues/1401#issuecomment-786384070)
+When you first enter the channel, type:
+`present+`
+
+Commands anyone can use:
+```
+q+  (to add yourself to the speaker queue)
+q+ to say [what you want to say]  (to add yourself to the speaker queue with reminder message)
+ack me  (take me off the q; or can use q-)
+ack ja  (take james off the q; or can use q- ja)
+q?  (to find out if anyone is on the q)
+
+s/old text/new text/  (to replace something that was typed. Make sure "old text" is unique)
+s/old text//  (to delete "old text")
+
+/me message  (sends message to channel without adding to minutes. Adds * prefix, i.e.)
+* joanie says Zakim is named after a bridge in Boston and pronounced Zay come
+```
+
+These are the commands to set up a meeting
+```
+meeting: ARIA WG
+chair: JamesNurthen
+agenda?  (to list the agenda items - chair usually enters agenda using `agenda+ item`)
+```
+
+Scribing commands used during the call:
+```
+scribe: [your nickname]
+zakim, next item
+zakim, close this item   (might need this if we try to move to the next item too soon)
+zakim, take up item 6   (might need this to go directly to a specific item)
+
+action: name - sentence about action item  (we only rarely add action items)
+topic: topic name  (ARIA WG doesn't use this, but APG does)
+github: [github issue url]  (adds current topic discussion to github issue)
+agenda+ [topic to add to agenda]
+```
+
+When scribing conversation, tab key will (usually) complete the person's name after 2 or 3 letters, for example:
+```
+ja[tab] completes to jamesn: 
+jc[tab] completes to jcraig: 
+```
+
+Scribing commands used at the end of the call to prepare the minutes:
+```
+zakim, who is on the call?  (list of everybody who typed present+)
+zakim, who is here?  (list of everyone on IRC, even those who forgot to type present+, and some who never log off)
+present+ name  (if anyone who was on the call is missing from the list)
+regrets+ name  (for anyone who sent regrets)
+rrsagent, make minutes
```

Comment:
@jnurthen introduced me to `zakim, end meeting` which triggered rssagent as well. Maybe that's better to use as default instructions; we could mention `rssagent` as fallback when there was additional chat after `end meeting`

Reply:
yes - re-running rrsagent is still useful to know if you need to fix the present list or perform other corrections.

Reply:
can probably drop the other zakim commands as end meeting executes them anyway